Classes/LanguageDetection.php

```diff
@@ -452,7 +475,7 @@ protected function getSysLanguages() {
 			}
 		} else {
 			$res = $this->getDB()->exec_SELECTquery(
-				'sys_language.uid, sys_language.title as isocode',
```

Comment:
Sorry for the late response.

We need to support TYPO3 6.2 and there is no such field available, so it would break the 6.2 functionality.